packages/flutter_tools/lib/src/devfs.dart

```diff
@@ -349,12 +349,16 @@ class UpdateFSReport {
   int get invalidatedSourcesCount => _invalidatedSourcesCount;
   int get syncedBytes => _syncedBytes;
 
+  // Only used for JavaScript compilation.
+  List<String> invalidatedModules;
```

Comment:
nit: is "module" part of Dart's terminology? Or is it referring to JS modules directly?

Reply:
These are JavaScript modules, updated doc comment